src/templates/template-common/requirements.txt

```diff
@@ -2,6 +2,9 @@ torch>=1.10.2
 torchvision>=0.11.3
 pytorch-ignite>=0.4.8
 pyyaml
+#::: if (['python-fire'].includes(it.argparser)) { :::#
+#:::= it.argparser :::#
+#::: } :::#
```

Comment:
to install python-fire we do `pip install fire` but requirements.txt shows:
```
...
python-fire
```
is it OK ?